crates/xilem_masonry/src/lib.rs

```diff
@@ -201,18 +201,41 @@ where
         EventLoopRunner::new(self.root_widget, window, event_loop, self.driver).run()
     }
 }
-pub trait MasonryView<State, Action = ()>: Send + 'static {
-    type Element: Widget + StoreInWidgetMut;
+
+pub trait ViewElement {
+    type Mut<'a>;
+    type Erased: ViewElement;
```

Comment:
Would be good to at least add some basic documentation to this trait. E.g. that this is type-erased and not necessarily a deleted element.